elasticsearch/docker-compose.yml

```diff
@@ -23,17 +23,17 @@ services:
     networks:
       - aether
 
-  elasticsearch-consumer:
-    extends:
-      file: docker-compose-base.yml
-      service: es-consumer-base
-    environment:
-      ELASTICSEARCH_INSTANCE_URLS: "elasticsearch"
-      BOOTSTRAP_SERVERS: "kafka:29092"
-      CONSUMER_PORT: 9099
-      ES_CONSUMER_CONFIG_PATH: "/code/conf/consumer/consumer.json"
-      ES_CONSUMER_KAFKA_CONFIG_PATH: "/code/conf/consumer/kafka.json"
-    links:
-      - elasticsearch
-    networks:
-      - aether
+  # elasticsearch-consumer:
```

Comment:
This block will be uncommented once you finish MTing the Consumer, right?

Reply:
Yessir

Reply:
Yessir